### PR TITLE
Update ci

### PR DESCRIPTION
**Pull request by lufernando100**
Update CI files

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        run: npm ci
+
+      - name: Code style
+        run: npm run lint
+
+      - name: UI E2E Tests
+        uses: cypress-io/github-action@v4
+        with:
+          browser: chrome
+          headed: true
```
Comment (Rendxn) [inline .github/workflows/main.yml]: Remove `headed`, since by default the Cypress Action runs it in headless mode (without an actual UI of the browser).

**Review by Rendxn — CHANGES_REQUESTED** [review]

**Comment by Rendxn** [comment]
Are you merging main onto the feature branches or why is there this many conflicts?

File: .github/workflows/main.yml (hunk 1 above)
Comment (lufernando100) [reply .github/workflows/main.yml]: It was resolved

**Review by Rendxn — CHANGES_REQUESTED** [review]
Remove duplicate cypress entry in devDependencies.

**Review by Rendxn — APPROVED** [review]

**Review by JuanDelahozMiranda — APPROVED** [review]

**Pull request merged**